src/main/java/esi/backend/service/RequestService.java

```diff
@@ -16,6 +19,8 @@ public class RequestService {
 
```

Comment:
Unfortunately couldn't comment on unchanged lines but in method `getRequest(UUID id)` I suggest getting rid of the `Optional<Request>` return type and handling the `Optional` case in the method. The new return type could be  `ResponseEntity<Request>`. You can check my PR #7 class `InvoiceService` method `getInvoice(UUID id)` for inspo on how to get rid of `Option` (has yet to change to the `ResponseEntity` layout) and Lana's PR #6 class `RequestService` on how to get the `ResponseEntity<Request>` type.

Reply:
ResponseEntity added with [976c758](https://github.com/ESI2022-team-e/backend/pull/5/commits/976c758bb4d379e88f538fc72116b3bad85d5d9e)